tests/components/brother/test_config_flow.py

```diff
@@ -0,0 +1,116 @@
+"""Define tests for the Brother Printer config flow."""
+import json
+
+from asynctest import patch
+from brother import SnmpError, UnsupportedModel
+
+from homeassistant import data_entry_flow
+from homeassistant.components.brother import config_flow
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_TYPE
+
+from tests.common import load_fixture
+
+CONFIG = {
+    CONF_HOST: "localhost",
+    CONF_NAME: "Printer",
+    CONF_TYPE: "laser",
+}
+
+
+async def test_show_form(hass):
+    """Test that the form is served with no input."""
+    flow = config_flow.BrotherConfigFlow()
+    flow.hass = hass
+
+    result = await flow.async_step_user(user_input=None)
```

Comment:
It's better to use the core interfaces to set up the test.

```py
result = await hass.config_entries.flow.async_init(
    DOMAIN, context={"source": "user"}
)
```

https://github.com/home-assistant/home-assistant/blob/63347ebeb5e84b5d1fbeeb25c5017831a2ff40bc/homeassistant/data_entry_flow.py#L94-L96